test/previews/primer/alpha/tooltip_preview.rb

```diff
@@ -14,6 +14,22 @@ def default(type: :description, direction: :s, tooltip_text: "Tooltip text")
         end
       end
 
+      # @param tooltip_text text
+      def label_tooltip_on_button_with_existing_labelledby(type: :label, direction: :s, tooltip_text: "Tooltip text")
+        render(Primer::ButtonComponent.new(id: "button-with-existing-label", "aria-labelledby": "existing-label-id")) do |c|
+          c.tooltip(text: tooltip_text, type: type, direction: direction)
+          "Button"
+        end
+      end
+
+      # @param tooltip_text text
```

Comment:
> Please review the changes carefully. The thing to watch out for most are layout issues but I don't believe we should encounter it given sr-only should not take up space.

I wonder if we could add a mock preview for an edge case where it could potentially break layout? Maybe use the bugs from last week as a reference. There's a way to spec out some basic HTML for a preview (you can see some examples of that with Autocomplete I believe). What do you think?

Reply:
That would be ideal! I was having trouble figuring out how to have multiple elements/components within a PVC preview. I'll have to take a look at `Autocomplete`. Mind if I tag you for extra eyes once I figure it out?

Reply:
@khiga8 here's an example: https://github.com/primer/view_components/tree/main/test/previews/primer/beta/auto_complete_item_preview

Yes please let me know how I can help!

Reply:
Added a preview and a test in https://github.com/primer/view_components/pull/1307/commits/162afb0ae34b0d0a53c2b9fb48c043b0a4a9abf8 ✅.

(FYI I couldn't get the background color change that we talked about over slack)